### DISCUSSION TITLE
Wyze Cam V3 stuttering and lagging

### DISCUSSION THREAD

**Discussion by jkfde** [blakeblackshear/frigate#2820] [Ask A Question]
So I set up frigate on a Pi4, Intel NUC and a full fledged desktop PC with intent to record the outdoor cameras and use live view in browser.

I'm running Ubuntu Server 20.04 LTS with everything in docker

So for recording, it works great I can playback in realtime and view it. But Live view is limited to 5 FPS and no matter what settings I change it always is stuck at 5 FPS. This causes the live view to look choppy and it loses connection and reconnects a lot.

I am using hardware acceleration. Without detection.

Should also note the RTMP stream is flawless. It's the RTSP stream that has issues

Wondering if this is an issue with Wyze RTSP firmware, a software misconfiguration or the network can't keep up?

**Comment by NickM-27** [comment]
The frigate live view uses the detect steam which is 5 fps. It would not be great to change that as that would put more strain on the CPU doing more detections. It also shouldn't cause actual disconnects / errors. 

I have a Wyze Cam V2 along with 2 Amcrest cams. The Amcrest cams are solid with no errors but the wyze always fills the frigate log with errors. Unfortunately their RTSP implementation just seems to have issues

**Reply by jkfde** [reply]
hmm even if I change the detect FPS, its still stuck at "displaying at 5fps" when debug shows like 8-10. I think I'll chalk it up to poor RTSP support on Wyzes end. Do you know if the docker bridge works any better?

**Reply by blakeblackshear** [reply]
The web UI in frigate will not go above 5fps for the debug view. The live view should go up to 10. Recordings will be at the original frame rate.

**Comment by C2BB** [comment]
Many folks are gravitating toward the wz_mini_hacks project which includes RTSP among many other great tweaks and it allows you to  leave the original firmware intact and functional.

https://github.com/gtxaspec/wz_mini_hacks

**Reply by ljmerza** [reply]
is the rtsp actually better though?

**Reply by C2BB** [reply]
it's been working flawlessly for me for many months now. I have frigate configured to connect with UDP.

**Reply by ashatsea1** [reply]
> it's been working flawlessly for me for many months now. I have frigate configured to connect with UDP.

Same here, no issues, and I'm about to add a fourth camera.

**Reply by alej908** [reply]
It can also make the wyze operate over ethernet with a usb to ethernet adapter. I'm using Wyze Mini Hacks, bumped up the FPS, the bitrate, and have it in standalone mode so it doesn't connect with Wyze. Its amazing!  The rtsp was great over wifi with the minihacks, but took it to the next level with usb to ethernet.

**Reply by pjuhl2313** [reply]
@alej908 do you happen to have a rundown of how you did all that?
